.ci_support/linux64.yaml

```diff
@@ -2,9 +2,23 @@ c_compiler:
   - gcc
 cxx_compiler:
   - gxx
-fortran_compiler:
-  - gfortran
 channel_sources:
   - conda-forge,defaults
+cgo_compiler:
+  - go-cgo
+cuda_compiler:
+  - nvcc
+cuda_compiler_version:
+  - None
+  #- 9.2
 docker_image:
   - condaforge/linux-anvil-comp7
+  #- condaforge/linux-anvil-cuda:9.2
+zip_keys:
+  - - cuda_compiler_version
+    - docker_image
+boost_cpp:
+  - 1.72
+pin_run_as_build:
+  boost-cpp:
+    max_pin: x.x.x
```

Comment:
Please changes files outside of the recipes dir in a separate PR

Reply:
Also, it seems like these setting could go in conda-build-config, perhaps?

Reply:
The typical approach to this is to try to delete some things from the Azure image that we're not using. This could free up some space.

Reply:
Moved linux64.yaml changes under the recipe.